templates/cloud/index.html

```diff
@@ -1,526 +1,241 @@
+  <div class="row u-equal-height">
+    <div class="col-5">
+      <p><b>优点：</b></p>
       <ul class="p-list">
-        <li class="p-list__item is-ticked">共同设计符合您要求的架构</li>
-        <li class="p-list__item is-ticked">我们在您选择的数据中心构建OpenStack，并按照服务级别协议(SLA)开展运营</li>
-        <li class="p-list__item is-ticked">当您的团队准备好接手OpenStack时，我们会将其移交给您</li>
+        <li class="p-list__item is-ticked">即刻访问基础设施即服务 (IaaS) 可加快企业/组织的产品和服务的上市时间。</li>
+        <li class="p-list__item is-ticked">公有云提供了几乎无限制地扩展资源消耗的能力。</li>
       </ul>
-      <p><a class="p-link--external" href="https://www.ubuntu.com/cloud/managed-cloud">让Canonical构建、运营并移交您的OpenStack</a></p>
     </div>
-    <div class="col-6">
-      <ul class="p-inline-images">
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/01ed776c-veritas.svg" alt="Veritas">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/66bc9135-logo-lexisnexis.png" alt="Lexis Nexis">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/25de1877-Tele2.svg" alt="Tele2">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/d0b7aa28-logo-deutsche-telekom.png" alt="Deutsche Telekom">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/481063cd-pccw.svg" alt="PCCW logo">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/97a050d0-Groupe_Cogeco_logo.svg" alt="Cogeco Datacenters">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/2353fc84-fibernet.svg" alt="Fibernet">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/d56d0b96-Biznet-Gio.png" alt="Biznet Gio">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/ff4e18ac-mimedia_logo_lrg.png" alt="Mimedia">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/507a0979-Voidbridge.jpg" alt="Voidbridge">
-        </li>
-        <li class="p-inline-images__item p-inline-images__item--smaller">
-          <img src="https://assets.ubuntu.com/v1/2e38f398-uct.svg" alt="UCT">
-        </li>
+    <div class="col-5">
+      <p><b>缺点：</b></p>
+      <ul class="p-list">
+        <li class="p-list__item is-crossed">虽然公有云的总体成本看起来很有吸引力，但隐藏的附属成本通常使其成为云架构的昂贵选择。</li>
```

Comment:
Can you add the `is-crossed`?

Reply:
Yes, it's missing in the style.scss, added.